src/QuickMarcEditor/QuickMarcEditor.test.js

```diff
@@ -895,6 +895,30 @@ describe('Given QuickMarcEditor', () => {
     });
   });
 
+  describe('when saving form without validation warnings or errors', () => {
```

Comment:
```suggestion
  describe('when saving form without validation warnings and errors', () => {
```

Reply:
In English you can say "without X or Y" meaning "without X and without Y". A little weird, but that's how it is.
Interestingly, it works the same way as De Morgan's law in boolean logic, where "not (A or B) = (not A) and (not B)"

Reply:
Wow, I haven't thought about De Morgan since grad school!